nevergrad/optimization/optimizerlib.py

```diff
@@ -2055,6 +2078,114 @@ def _select_optimizer_cls(self) -> base.OptCls:
         return optimClass
 
 
+ProgD13 = ConfSplitOptimizer(num_optims=13, progressive=True,
+                             multivariate_optimizer=OptimisticDiscreteOnePlusOne).set_name("ProgD13", register=True)
+
+RecombiningOptimisticNoisyDiscreteOnePlusOne = ParametrizedOnePlusOne(crossover=True, mutation="discrete",
+                                                                      noise_handling="optimistic").set_name(
+    "RecombiningOptimisticNoisyDiscreteOnePlusOne", register=True)
+
+
+@registry.register
+class NGOpt8(base.Optimizer):
```

Comment:
please derive from NGOptBase or another version (NGOpt4?), and follow the new way of implementating this, I spent a lot of time trying to structurate them a bit better

Reply:
I suggest to keep only version at the end, but we need to discuss with @teytaud .